src/app/types/Server.ts

```diff
@@ -406,7 +406,10 @@ export enum ServerAction {
   BILLING_FORCE_SYNCHRONIZE_USER_INVOICES = 'BillingForceSynchronizeUserInvoices',
   BILLING_DOWNLOAD_INVOICE = 'BillingDownloadInvoice',
   BILLING_CREATE_TRANSACTION_INVOICE = 'BillingCreateTransactionInvoice',
+  BILLING_SETUP_PAYMENT_METHOD = 'BillingSetupPaymentMethod',
```

Comment:
Setup or Create?
You can also use the new REST endpoint (check with Claude)
Then the route is in ServerRoute and starts with REST_
Loan knows this very well.

Reply:
I was wondering about that. What is the best approach as it does the same right ?

Reply:
talking about the REST_
And this is Setup on purpose as it not only create a payment method